code/modules/surgery/organs/tongue.dm

```diff
@@ -190,9 +223,43 @@
 	say_mod = "states"
 	attack_verb = list("beeped", "booped")
 	taste_sensitivity = 25 // not as good as an organic tongue
+	maxHealth = 100 //RoboTongue!
 
 /obj/item/organ/tongue/robot/can_speak_in_language(datum/language/dt)
 	. = TRUE // THE MAGIC OF ELECTRONICS
 
 /obj/item/organ/tongue/robot/get_spans()
 	return ..() | SPAN_ROBOT
+
+//FermiChem
+/obj/item/organ/tongue/OwO
```

Comment:
please do not put this stuff in typepaths, the owo/memes are already bad enough when we hvae all of it in comments but typepaths is a bit far.

Reply:
Sorry Kevin uwu.

Reply:
Do you want me to remove/edit the description?